core/src/servers.rs

```diff
@@ -434,5 +463,40 @@ pub async fn create_verifiers_and_operators_grpc(
     .await
     .unwrap();
 
-    (verifier_endpoints, operator_endpoints, aggregator)
+    let port = port + 1;
+    println!("Watchtower start port: {}", port);
+    let wathctower_futures = (0..number_of_watchtowers)
+        .map(|i| {
+            let verifier_endpoints = verifier_endpoints.clone();
+            let operator_endpoints = operator_endpoints.clone();
+            let verifier_configs = verifier_configs.clone();
+
+            create_watchtower_grpc_server(BridgeConfig {
+                port: port + i as u16,
+                verifier_endpoints: Some(
```

Comment:
Watchtower server does not need verifier endpoints and operator endpoints

Reply:
Fixed in https://github.com/chainwayxyz/clementine/commit/1f907464d6118da689cdc1e91759010445a713aa

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/373/commits/5b596ea9e6de607cc8fe5aa2bf0dd81d50f6fa6a